repos/system_upgrade/common/actors/checkbootavailspace/actor.py

```diff
@@ -1,8 +1,5 @@
 from leapp.actors import Actor
-from leapp.libraries.actor.checkbootavailspace import (
-    check_avail_space_on_boot,
-    get_avail_bytes_on_boot,
-)
+from leapp.libraries.actor.checkbootavailspace import check_avail_space_on_boot, get_avail_bytes_on_boot
```

Comment:
in this case, the import is sorted already, why to do this change? I do not see so big benefit to change? I do not see this anyhow better.

Reply:
Per current configuration multi line imports are used only if single line import is not possible. I'll see if it's possible to relax the rule.

Reply:
Unfortunately, I could not figure out how to do this (keep both styles even when imports can be made on a single line without violating max line length setting).
Are you ok to keep the current logic if we go with a gradual rule introduction https://github.com/oamg/leapp-repository/pull/770/ ?